src/Database/SqlServer/Generator.hs

```diff
@@ -46,6 +46,14 @@ data GenerateOptions = GenerateOptions
   , excludeTypes :: RenderOptions
   }
 
+defaultGenerateOptions :: GenerateOptions
```

Comment:
Good point.  What I really want is to only define a default for the RenderOptions and not do anything with the others. I've committed a change, what do you think? The con to my change is that you loose the ability (as far as I am aware) to specify the arguments by name.  I don't know if we care?